draft-ietf-quic-multipath.md

```diff
@@ -1399,6 +1399,17 @@ it cannot allocate sufficient resources to maintain it. This is due to
 limitations in the transport layer's capacity. This error indicates that
 resource constraints prevent the continuation of the path.
 
+UNSTABLE_INTERFACE (TBD-11): The endpoint is abandoning the path because
+the used interface is considered to be unstable. This condition can occur, e.g.,
+due to a weak wireless signal or frequent mobility events.
```

Comment:
What does "frequent mobility events" mean here? Maybe consider making this more clear.

Reply:
This is e.g. when you walk constantly in and out of coverage. However, these are really just supposed to be examples, so I don't think we need to go too much into detail. Which part is not clear? Do you have an alternative wording that you think would be more clear?

Reply:
```suggestion
due to a weak wireless signal or frequent handover events during high-speed mobility.
```